packages/dropdown/src/react/index.js

```diff
@@ -109,6 +110,9 @@ class Dropdown extends React.Component {
     if (typeof this.props.onBlur === 'function') this.props.onBlur(evt)
   }
   handleKeyDown(evt) {
+    if (evt.key === 'ArrowDown' || evt.key === ' ' || evt.key === 'Enter') {
```

Comment:
All 3 because this is where I want to discover "is a keyboarder!" but the space and enter keys also trigger onClick

Reply:
what does `evt.key === ' '` check for? is it the space key and is there no `Space` string? asking for a friend

Reply:
Yes, it's for space.  In my experimenting there was no `Space` string, just 1 char space string.